Sources/Plasma/PubUtilLib/plInputCore/plAvatarInputInterface.cpp

```diff
@@ -916,6 +929,8 @@ bool plAvatarInputInterface::InterpretInputEvent( plInputEventMsg *pMsg )
                         continue;
                     if (mouseInfo->fControlFlags & kControlFlagMiddleButtonUp && !(pMouseMsg->GetButton() == kMiddleButtonUp))
                         continue;
+                    if (mouseInfo->fControlFlags & kControlFlagWheel && !(pMouseMsg->GetButton() == kWheelPos | pMouseMsg->GetButton() == kWheelNeg))
```

Comment:
MSVC is warning about this. Did you mean:
```suggestion
                    if (mouseInfo->fControlFlags & kControlFlagWheel && !(pMouseMsg->GetButton() == kWheelPos || pMouseMsg->GetButton() == kWheelNeg))
```